src/views/Submit.vue

```diff
@@ -164,6 +179,7 @@ export default {
 			answers: {},
 			loading: false,
 			success: false,
+			submitForm: false,
```

Comment:
```suggestion
			/** Submit state of the form, true if changes are already submitted */
			submitForm: false,
```

Some kind of documentation would be good as the name is not that self explanatory.